nautobot/dcim/templates/dcim/device.html

```diff
@@ -142,8 +142,13 @@
                                                     {{ object.primary_ip4|hyperlinked_object|add_html_id:"ipv4" }}
                                                     {% if object.primary_ip4.nat_inside %}
                                                         <span>(NAT for {{ object.primary_ip4.nat_inside.address.ip }})</span>
-                                                    {% elif object.primary_ip4.nat_outside %}
-                                                        <span>(NAT: {{ object.primary_ip4.nat_outside.address.ip }})</span>
+                                                    {% elif object.primary_ip4.nat_outside_list.exists %}
+                                                        <br /><span>NAT: </span>
+                                                        <ul>
+                                                            {% for nat in object.primary_ip6.nat_outside_list.all %}
+                                                                <li>{{ nat.address.ip }}</li>
```

Comment:
Should this also be hyperlinked?

Reply:
I dont think so, It was not initially an hyperlink